src/components/Shop/ShopWindow/index.tsx

```diff
@@ -1,24 +1,24 @@
-import React, { FC } from "react";
+import React, { useContext, FC } from "react";
 import styled from "styled-components";
 import ShopItem from "./ShopItem";
 import { OnlineOrange, OffWhite } from "utility/style";
+import { StateContext } from "state/state";
+import { addToCart } from "state/actions";
+import { Product } from "types/inventory";
+
 const ShopWindow: FC = () => {
-  return (
-    <Container>
-      <ShopItem />
-      <ShopItem />
-      <ShopItem />
-      <ShopItem />
-      <ShopItem />
-      <ShopItem />
-      <ShopItem />
-      <ShopItem />
-      <ShopItem />
-      <ShopItem />
-      <ShopItem />
-      <ShopItem />
-    </Container>
-  );
+  const { state, dispatch } = useContext(StateContext);
+
+  const addItem = (id: number) => dispatch(addToCart(id));
+
+  const shopItems = state.inventory.map((item) => {
```

Comment:
Change the `==`'s to `===`.
Is there a difference between those to if checks? If not, maybe just do it in one check. `item.category.name === state.category || state.category === 'Alt'`